### ISSUE TITLE
Update translation files before releasing 2.0.0

### ISSUE THREAD

**Issue by shu-mutou** [kubernetes/dashboard#4259]
##### What would you like to be added
Before releasing 2.0.0, update translation files, i.e. `i18n/[locale]/messages.[locale].xlf`, for new releases.

##### Why is this needed
Since the last translation, new commits for frontend have been made and translation files have updated.

##### Comments
To add or update translation file, see [this document](https://github.com/kubernetes/dashboard/blob/master/docs/developer/internationalization.md).

##### Translation files to be updated (2020-1-22)
- [x] fr: done by #4828
- [x] ja: done by #4784
- [x] ko: done by #4819, #4859
- [x] zh (Simplified Chinese): ~#4806~ #4856
- zh-cn (Simplified Chinese): Linked to `zh` #4814
- zh-sg (Simplified Chinese): Linked to `zh` #4856
- [x] zh-tw (Traditional Chinese): #4815
- [x] zh-hk (Traditional Chinese): #4815

##### PRs that require translation
- ~Implement the front end of clusterrole #4622~ **_hold_**
- ~Enable to switch context in kubeconfig #4534~ **_hold, >2.0.0_**

##### String fix (2020-2-28)
- [x] In principle, additions or changes of strings that require further translation will not occur. #4911
  Above PRs that need translation would be putted off to next release.

##### Translation files to be merged (should be merged until 2020-3-6)
- [x] de: #4905 

##### Angular 9 support (starts at 2020-3-9, until 2020-3-13)
- [x] Use new locale id: #4969
  zh-cn -> zh-Hans, zh-sg -> remove, zh-tw -> zh-Hant, zh-hk -> zh-Hant-HK, zh -> remove

##### Translation files to be checked (starts at 2020-3-16, until 2020-3-20)
- [x] Final check of translation files by approver of each language.

##### Translation files to be checked (starts at 2020-3-23, until 2020-3-27)
- [x] Final check of translation files. (cleanup `status="new"` and update `linenumber`s) @shu-mutou #4969

**Comment by shu-mutou** [comment]
@feloy @shu-mutou @fuergaosi233 @seokho-son please follow this issue.
related #3812 #4173 #4063 #4174

**Comment by shu-mutou** [comment]
@feloy @fuergaosi233 @seokho-son BTW, if you would get permission to approve translation files, i.e. if directories for each languages with OWNERS file that includes you would be added, would you guys be happy?

**Comment by feloy** [comment]
Yes, that's ok for me. Thanks

**Comment by seokho-son** [comment]
Sure :) Thank you.

**Comment by shu-mutou** [comment]
Thanks guys, I will submit another issue for it. Then let's talk about it. 😉

**Comment by jeefy** [comment]
It seems like most of the work is complete. Should we close this issue or is there still some remaining effort?

**Comment by shu-mutou** [comment]
Hi @jeefy. Including how to watch updates, our i18n framework has been completed already. But we need to confirm each i18n teams whether i18n files should update before 2.0.0 release. This issue aims to inform them time to release 2.0.0. If the time comes, could maintainers let i18n teams know here?

**Comment by maciaszczykm** [comment]
@feloy @shu-mutou @fuergaosi233 @seokho-son Could you please take a look on translations and add missing ones? We're about to release v2.

**Comment by seokho-son** [comment]
> @feloy @shu-mutou @fuergaosi233 @seokho-son Could you please take a look on translations and add missing ones? We're about to release v2.

@maciaszczykm  Thank you for the notification. I will check it for Korean localization. When is the target date for release?

**Comment by maciaszczykm** [comment]
We want to do it as soon as possible. We need some more tests and to merge 3-4 existing pull requests. Translations are important part too.

**Comment by shu-mutou** [comment]
I found some elements to be translated. I'll fix them ASAP. Please be patient.
And I filed a issue for some untranslatable elements as feature request. #4699

**Comment by maciaszczykm** [comment]
@pengx17 @riverzhang @zouyee @fuergaosi233 Hello guys, you were helping us with Chinese translations before, could you take a look on current state of the files? We were having some issues with finding an agreement (https://github.com/kubernetes/dashboard/pull/4694) and any additional opinion could help us.

**Comment by shu-mutou** [comment]
@maciaszczykm We have updated translation for all of implemented language. Although we have discussion(#4705) that should be done widely, i.e. with SIG-docs, WG-UX or someplaces, it's enough for now, I think. 😉

**Comment by maciaszczykm** [comment]
Thank you a lot.

**Comment by shu-mutou** [comment]
Updated situations for each locales. We need to update translations again.

**Comment by shu-mutou** [comment]
@feloy @seokho-son Could you create PR for updating translations?

**Comment by shu-mutou** [comment]
We should do final check this before new release. I'll do that at mid March.

**Comment by shu-mutou** [comment]
Due to String Freeze (from 2020-2-28), PRs for new feature that needs translation, i.e. #4534 and #4622, will be freezed until new release (2020-3-30).

**Comment by shu-mutou** [comment]
@feloy @seokho-son @gochist @hwdef @tanjunchen @chenrui333 @zehuaiWANG 
We need to update translations. Could you check followings?
* Fix CRD details switch #5016
* Add option to disable access denied notifications #5004

**Comment by hwdef** [comment]
@shu-mutou 
I will update these. Thank you for your reminder.
